### ISSUE TITLE
Add print-object to rehearsal

### ISSUE THREAD

**Issue by rettinghaus** [w3c/musicxml#511]
Without the `print-object` attribute available on `<rehearsal>` it is not possible to carry invisible structural information about a piece.

**Comment by mdgood** [comment]
That's what the [`<grouping>`](https://www.w3.org/2021/06/musicxml40/musicxml-reference/elements/grouping/) element is for.

**Comment by mscuthbert** [comment]
I agree with Michael. But I also think that it wouldn't harm MusicXML or editors to have this option, since a score might encode two sets of rehearsal marks representing two different editions and digital editors might want to be able to switch between displaying one or the other. I've generally felt that there is little harm for encoding to suppose that any element might be there for possible display later but not want to be displayed at the moment. Making five hundred musical excerpts in the past 4 months has made me more conscious of needs like this. (A four measure excerpt with rehearsal mark on m 2 doesn't need it displayed for instance)

**Comment by rettinghaus** [comment]
So you mean something like 

```xml
<grouping type="single">
   <feature type="rehearsal">B</feature>
</grouping>
```

I think that would work. But then you'd lose all the layout information from the original rehearsal mark. And as @mscuthbert said, it wouldn't harm MusicXML, I guess.
